moxy/org.eclipse.persistence.moxy/src/main/java/module-info.java

```diff
@@ -43,6 +43,7 @@
     exports org.eclipse.persistence.jaxb.rs;
     exports org.eclipse.persistence.jaxb.xmlmodel;
     opens org.eclipse.persistence.jaxb.xmlmodel to jakarta.xml.bind;
+    opens org.eclipse.persistence.jaxb.rs; //required for MOXyJSON provider injection
```

Comment:
Just a question. Is chance to specify `to <module name>` there?

Reply:
I would rather not. As soon as it is the public API, it shall be open for reflection to anyone. If the ``to`` part is specified, only that framework will be allowed to inject the provider. What about CDI then?